GliaWidgets/ViewModel/EngagementViewModel.swift

```diff
@@ -147,6 +165,7 @@ class EngagementViewModel {
                         }
                     )
                 )
+                isEngagementEndedByOperator = true
```

Comment:
Q: We change the `isEngagementEndedByVisitor` on the confirmation step after the session is ended, but here is before the alert has been presented. Is it correct behavior or do we need to restore consistency and do `isEngagementEndedByOperator = true` after `self?.endSession()` also?

```swift
confirmed: { [weak self] in
    self?.endSession()
    self?.isEngagementEndedByVisitor = true
}
```

Reply:
when visiotr taps on button "X" (close) and confirms that he wants to end engagement (=taps on "YES") then isEngagementEndedByVisitor = true. But in that case I thought that i don't need to stick isEngagementEndedByOperator variable to action when visitor taps OK button in alert. I just need to know when engagement is ended by operator. And if it was ended by operator i let View Controller to know that it was ended by operator